modules/cli/pages/cbstats/cbstats-memory.adoc

```diff
@@ -15,120 +15,96 @@ cbstats [host]:11210 memory
-
-| `total_free_mapped_bytes`
-| Number of bytes in free, mapped pages.
-
-| `total_free_unmapped_bytes`
-| Number of bytes in free, unmapped pages.
-
-| `total_heap_bytes`
-| Number of of bytes in a heap.
 |===
 
-== Options
-
-None
-
 == Example
 
-Request
+*Request*
 
 ----
-cbstats 10.5.2.54:11210 memory
+/opt/couchbase/bin/cbstats -u Administrator -p password -b travel-sample localhost:11210 memory
 ----
 
 *Response*
 
 ----
- bytes:                               23716184
- ep_blob_num:                         0
- ep_blob_overhead:                    0
- ep_item_num:                         1364
- ep_kv_size:                          0
- ep_max_size:                         524288000
- ep_mem_high_wat:                     445644800
- ep_mem_low_wat:                      393216000
- ep_mem_tracker_enabled:              true
- ep_oom_errors:                       0
- ep_overhead:                         18344996
- ep_storedval_num:                    0
- ep_storedval_overhead:               0
- ep_storedval_size:                   0
- ep_tmp_oom_errors:                   0
- ep_value_size:                       0
- mem_used:                            23716184
- tcmalloc_current_thread_cache_bytes: 18642072
- tcmalloc_max_thread_cache_bytes:     33554432
- total_allocated_bytes:               211297528
- total_fragmentation_bytes:           33069832
- total_free_mapped_bytes:             5283840
- total_free_unmapped_bytes:           7684096
- total_heap_bytes:                    257335296
+bytes:                              68505152
+ ep_arena:allocated:                 69533504
+ ep_arena:arena:                     2
+ ep_arena:base:                      958544
+ ep_arena:fragmentation_size:        1798336
+ ep_arena:internal:                  0
+ ep_arena:large.allocated:           1146880
+ ep_arena:mapped:                    72466432
+ ep_arena:resident:                  71331840
+ ep_arena:retained:                  10371072
+ ep_arena:small.allocated:           68386624
+ ep_arena_global:allocated:          10353984
+ ep_arena_global:arena:              0
+ ep_arena_global:base:               4922096
+ ep_arena_global:fragmentation_size: 6808256
+ ep_arena_global:internal:           1153648
+ ep_arena_global:large.allocated:    5197824
+ ep_arena_global:mapped:             24813568
+ ep_arena_global:resident:           17162240
+ ep_arena_global:retained:           2973696
+ ep_arena_global:small.allocated:    5156160
+ ep_blob_num:                        64824
+ ep_blob_overhead:                   4247418
+ ep_item_num:                        3584
+ ep_kv_size:                         48627966
+ ep_max_size:                        209715200
+ ep_mem_high_wat:                    178257920
+ ep_mem_high_wat_percent:            0.85
+ ep_mem_low_wat:                     157286400
+ ep_mem_low_wat_percent:             0.75
+ ep_mem_used_primary:                68505152
+ ep_mem_used_secondary:              0
```

Comment:
Might be worth adding these - they're reasonably straightforward for a user to grasp.

primary:

If this is a magma bucket, this is the memory usage of the bucket excluding magma. Else, this is equivalent to mem_used / bytes.

secondary:

If this is a magma bucket, this is the memory usage of the storage layer. Else, this is unused and will be zero.